### ISSUE TITLE
FromUtc Pipe

### ISSUE THREAD

**Issue by fshin123** [urish/angular2-moment#179]
I am using version 1.7.0 and noticed that the FromUTC pipe is not being exported by this module.  Is this an oversight when this pipe was added with this release?  Or am I missing something?

Thanks!
Frank

**Comment by fshin123** [comment]
FYI-- this was fixed with commit: 25f3b6e55b18585ba1b2a4afd6ad47dd72fa637b.

**Comment by gigadie** [comment]
Hi,

but I also noticed the from-utc pipe is missing from the index.js, therefore importing it as:
```javascript
import { FromUtcPipe } from 'angular2-moment';
```
unfortunately throws an error.
```shell
error TS2305: Module '"[...]node_modules/angular2-moment/index"' has no exported member 'FromUtcPipe'.
```

Thanks
Diego

**Comment by urish** [comment]
Released as 1.7.1
